### PR TITLE
Removed sinon-timers from the build. refs #811

### PR DESCRIPTION
**Pull request by grassator**
I'm not sure if this is all I have to do, but it's a start.

### CODE REVIEW

**Comment by mroderick** [comment]
Nice! I'll have to find some time to check it out and run it locally.

File: build [hunk 1]
```diff
@@ -109,20 +109,11 @@ Dir.chdir(File.dirname(__FILE__)) do
...
-  add_license("pkg/sinon-timers#{version_string}.js", version)
-  FileUtils.cp("lib/sinon/util/timers_ie.js", "pkg/sinon-timers-ie#{version_string}.js")
-  add_license("pkg/sinon-timers-ie#{version_string}.js", version)
-
   merger = Juicer::Merger::JavaScriptMerger.new
   merger << "lib/sinon/util/fake_server_with_clock.js"
   merger.save("pkg/sinon-server#{version_string}.js")
   add_license("pkg/sinon-server#{version_string}.js", version)
   FileUtils.cp(output, 'pkg/sinon.js')
-  FileUtils.cp("pkg/sinon-ie#{version_string}.js", 'pkg/sinon-ie.js')
```
Comment (mroderick) [inline build]: We probably want to keep making `sinon-ie.js` and `sinon-server.js` for a little while longer.

**Comment by grassator** [comment]
@mroderick sorry about that, now should be ok.

**Comment by mroderick** [comment]
:+1: could you squash it down to just one commit before we merge it?

**Comment by grassator** [comment]
@mroderick squashed

**Pull request merged**